# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/validationGL32.cpp
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/validationGL32.cpp
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/angle/checkout/src/libANGLE/validationGL32.cpp	Fri Nov 08 17:06:00 2019 +0000
@@ -0,0 +1,95 @@
+//
+// Copyright 2019 The ANGLE Project Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+
+// validationGL32.cpp: Validation functions for OpenGL 3.2 entry point parameters
+
+#include "libANGLE/validationGL32_autogen.h"
+
+namespace gl
+{
+
+bool ValidateDrawElementsBaseVertex(Context *context,
+                                    GLenum mode,
+                                    GLsizei count,
+                                    GLenum type,
+                                    const void *indices,
+                                    GLint basevertex)
+{
+    return true;
+}
+
+bool ValidateDrawElementsInstancedBaseVertex(Context *context,
+                                             GLenum mode,
+                                             GLsizei count,
+                                             GLenum type,
+                                             const void *indices,
+                                             GLsizei instancecount,
+                                             GLint basevertex)
+{
+    return true;
+}
+
+bool ValidateDrawRangeElementsBaseVertex(Context *context,
+                                         GLenum mode,
+                                         GLuint start,
+                                         GLuint end,
+                                         GLsizei count,
+                                         GLenum type,
+                                         const void *indices,
+                                         GLint basevertex)
+{
+    return true;
+}
+
+bool ValidateFramebufferTexture(Context *context,
+                                GLenum target,
+                                GLenum attachment,
+                                GLuint texture,
+                                GLint level)
+{
+    return true;
+}
+
+bool ValidateMultiDrawElementsBaseVertex(Context *context,
+                                         GLenum mode,
+                                         const GLsizei *count,
+                                         GLenum type,
+                                         const void *const *indices,
+                                         GLsizei drawcount,
+                                         const GLint *basevertex)
+{
+    return true;
+}
+
+bool ValidateProvokingVertex(Context *context, ProvokingVertexConvention modePacked)
+{
+    return true;
+}
+
+bool ValidateTexImage2DMultisample(Context *context,
+                                   GLenum target,
+                                   GLsizei samples,
+                                   GLenum internalformat,
+                                   GLsizei width,
+                                   GLsizei height,
+                                   GLboolean fixedsamplelocations)
+{
+    return true;
+}
+
+bool ValidateTexImage3DMultisample(Context *context,
+                                   GLenum target,
+                                   GLsizei samples,
+                                   GLenum internalformat,
+                                   GLsizei width,
+                                   GLsizei height,
+                                   GLsizei depth,
+                                   GLboolean fixedsamplelocations)
+{
+    return true;
+}
+
+}  // namespace gl